esp/applications/esdl-sandbox/esp.yaml

```diff
@@ -0,0 +1,28 @@
+esp:
+   instance: esdl-sandbox
+   description: ESDL Sandbox
+   daliServers: mydali
+   loadDaliBindings: true
+   auth: ldap
+   tls: true
+   port: 8880
+   enableSEHMapping: true
+   httpConfigAccess: true
+   logLevel: 1
+   maxBacklogQueueSize: 200
+   portalurl: http://hpccsystems.com/download
+   logDir: "-"
+
+   tls_config:
+     certificate: /etc/HPCCSystems/certificates/{$instance}/server.crt
+     privatekey: /etc/HPCCSystems/certificates/{$instance}/private.key
+     passphrase: JHjdi6DDptMwYmJNTZsqjg==
```

Comment:
This makes me nervous!

Reply:
These are meant as (semi) self documenting placeholders for the end users real passphrase.  This simply decodes to "secure".  I can easily remove it. 
When I get back around to refactoring the mutual tls stuff I will replace these tls config entries with secrets.